kontrakter/felles/kvittering/1.0/Kvittering.schema.json

```diff
@@ -0,0 +1,68 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "$id": "felles/1.0/Kvittering.schema.json",
+  "description": "Schema som beskriver kvitteringer som skal utveksles på tvers av virksomhetene",
+  "additionalProperties": true,
+  "properties": {
+    "meldingsId": {
+      "type": "string",
+      "description": "Meldingsid for selve kvitteringen"
+    },
+    "meldingsIdRef": {
+      "type": "string",
+      "description": "Meldingsid for meldingen vi kvitterer"
+    },
+    "status": {
+      "$ref": "#/definitions/StatusType",
+      "description": "Status for meldingen vi kvitterer"
+    },
+    "beskrivelse": {
+      "type": "string",
+      "description": "Tekstlig beskrivelse. Brukes i forbindelse med statusene FEILET_AVSENDER og FEILET_MOTTAKER"
+    },
+    "saksRef": {
+		"type": "string",
+		"description": "Saksnummeret fra Lovisa. Vil kun være med fra domstolen dersom status er MOTTATT (altså kun etter at saken er opprettet"
+    }
+  },
+  "definitions": {
+    "StatusType": {
+      "description": "Liste over mulige statuser",      
+      "enum": [
+        "UNDER_MOTTAK",
+        "MOTTATT",
+        "FEILET_AVSENDER_TEKNINSK",
+        "FEILET_AVSENDER_MENNESKELIG",
+        "FEILET_MOTTAKER"
+      ],
+      "type": "string",
+      "options": [{
+            "value": "UNDER_MOTTAK",
+            "description": "Indikerer at meldingen er mottatt, men ikke ferdig prosessert inn i saksbehandlingssystem. Det kan fremdeles oppstå feil som gjør at avsender eller mottaker må starte feilsøking."
+        }, 
+        {
+          "value": "MOTTATT",
+            "description": "Indikerer at alt er ok. Ingen nye kvitteringer forventes. Avsender kan fjerne vedlegg fra filtjeneste når MOTTATT er mottatt."
+        },
+        {
+          "value": "FEILET_AVSENDER_TEKNISK",
+            "description": "Indikerer at det er en teknisk feil hos avsender. F.eks. feil format, vedlegg som ikke fins på STFP selv om det fins referanse i melding etc. Feltet 'beskrivelse' skal fylles ut med en feilmelding/exception. Det er forventet av _avsender_ å iverksette nødvendige tiltak for å sende melding på nytt. En slik status skal aldri sendes etter at en MOTTATT melding er sendt. Ingen nye kvitteringer forventes.",
+        },
+        {
+          "value": "FEILET_AVSENDER_MENNESKELIG",
+          "description": "Indikerer at det er en menneskelig feil hos avsender. F.eks. at forventet påtedning er avglemt. Feltet 'beskrivelse' skal fylles ut med en feilmelding skrevet av bruker. Det er forventet av _avsender_ iverksette nødvendige tiltak for å sende melding på nytt. En slik status skal aldri sendes etter at en MOTTATT melding er sendt. Ingen nye kvitteringer forventes."
+        },
+        {
+          "value": "FEILET_MOTTAKER",
```

Comment:
Trengs `FEILET_MOTTAKER`? I utgangspunktet ville jeg tenkt at det holder å sende "UNDER_MOTTAK". Det er uansett mottaker sitt ansvar hvis noe feiler på mottakersiden, så trenger man å opplyse omverdenenen? Jeg er redd for at det bare fører til forvirring :)

Reply:
Etter en diskusjon på tvers av virksomhetene kom vi fram til at vi ønsker å ta med dette feltet videre.
Begrunnelsen er at man ønsker å gi avsender en indikasjon på at en feil har oppstått hos mottaker og saken ligger hos dem. Dersom vi ikke gir denne statusen blir det fort telefoner/epost frem og tilbake om hvorfor avsender ikke har mottatt noe MOTTATT status.